src/views/default/type_components/datamodal/component.blade.php

```diff
@@ -6,13 +6,17 @@
 	<?php 
 		$datamodal_field = explode(',',$form['datamodal_columns'])[0];
 		$datamodal_value = DB::table($form['datamodal_table'])->where('id',$value)->first()->$datamodal_field;		
+
 	?>	
 	
 	<div id='{{$name}}' class="input-group">
 	  <input type="hidden" name="{{$name}}" class="input-id" value="{{$value}}">
       <input type="text" class="form-control input-label {{$form['required']?"required":""}}" value="{{$datamodal_value}}" readonly>
       <span class="input-group-btn">
-        <button class="btn btn-primary" onclick="showModal{{$name}}()" type="button"><i class='fa fa-search'></i> Browse Data</button>
+        <button class="btn btn-primary" onclick="showModal{{$name}}()" type="button"><i class='fa fa-search'></i> {{trans('crudbooster.datamodal_browse_data')}}</button>
+			<?php if(strlen($form['datamodal_module_name'])>1){ ?>	
+        <a class="btn btn-info"  href="../../{{$form['datamodal_module_name']}}" target="_blank"><i class='fa fa-edit'></i> {{$form['label']}}</a>
```

Comment:
I don't really understand about this line, can you tell me? what is it? and why using a relative path? thx